Extensions/ArtifactEngine/Engine/logger.ts

```diff
@@ -63,14 +63,25 @@ export class Logger {
         var skippedItems = fileTickets.filter(x => x.state == TicketState.Skipped);
         var failedItems = fileTickets.filter(x => x.state == TicketState.Failed);
 
+        var downloadSizeInBytes = 0;
+        var fileSizeInBytes = 0;
+
+        for (var ticket of fileTickets) {
+            downloadSizeInBytes += ticket.downloadSizeInBytes;
+            fileSizeInBytes += ticket.fileSizeInBytes;
+        }
+
+        var downloadSizeInMB = (downloadSizeInBytes / (1024*1024)).toFixed(3);
+
         var endTime = new Date();
-        var timeElapsed = (endTime.valueOf() - this.startTime.valueOf()) / 1000;
+        var downloadTime = (endTime.valueOf() - this.startTime.valueOf()) / 1000;
         console.log(
             "Total Files: " + fileTickets.length
             + ", Processed: " + processedItems.length
             + ", Skipped: " + skippedItems.length
             + ", Failed: " + failedItems.length
-            + ", Time elapsed: " + timeElapsed + "secs");
+            + ", Download time: " + downloadTime + "secs"
+            + ", Download size: " + downloadSizeInMB + "MB");
```

Comment:
one of these will be 0 based on whether it is download from server drop or file share?

Reply:
Yes, for file share. However, fileSizeInBytes (logged in telemetry) will always be non-zero.